docs/documentation/server_development/topics/themes.adoc

```diff
@@ -59,9 +59,8 @@ A theme consists of:
 * Scripts
 * Theme properties
 
-Unless you plan to replace every single page you should extend another theme. Most likely you will want to extend the {project_name} theme, but you could also
-consider extending the base theme if you are significantly changing the look and feel of the pages. The base theme primarily consists of HTML templates and
-message bundles, while the {project_name} theme primarily contains images and stylesheets.
+Unless you plan to replace every single page you should extend another theme. Most likely you will want to extend the some existing theme. Alternatively, if you intend to provide your own implementation of the admin or account console,
+consider extending the `base` theme. The `base` theme consists of a message bundle and therefore such implementation needs to start from scratch, including implementation of the main `index.ftl` Freemarker template, but it can leverage existing translations from the message bundle.
```

Comment:
This intentionally does not mention UiPageProvider SPI for extending admin console as that should be documented on its own.